webapp/public/javascripts/controllers.js

```diff
@@ -234,18 +246,24 @@ lunchrControllers.controller(
                                             labelClass: "labels",
                                             animation: google.maps.Animation.DROP
                                         },
-                                        click: function (e) {
-                                            var index = $scope.selectedPlaces.indexOf(e.$id);
+                                        click: function( ) {
+                                            var errorAlert = $('#error');
+                                            if(errorAlert.html()){
+                                                errorAlert.html("");
```

Comment:
Why are we checking to see if the html exists, then resetting it here? Doesn't errorAlert contain an error message to display that is being erased on this line?